packages/desktop-client/src/components/reports/CashFlow.js

```diff
@@ -33,7 +33,7 @@ function CashFlow() {
 
   const [allMonths, setAllMonths] = useState(null);
   const [start, setStart] = useState(
-    monthUtils.subMonths(monthUtils.currentMonth(), 30),
+    monthUtils.subMonths(monthUtils.currentMonth(), 5),
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏this is great, but would be even better if we would store this in local storage. And then re-use between all the reports.

Reply:
That might be a bit out of reach of my ability so it might take a bit longer for me to do. I'm happy for this to be merged in if you approve and I can try to do that in another PR.

Reply:
Sure thing!